dockstore/dockstore-ui2
# Change Nextflow Tower to Seqera

**Description**
Changes Nextflow Tower logo and text to a Seqera logo (that includes text).

On the web service side, note that there is a NEXTFLOW_TOWER platform partner that we should update (deprecate and add a new one?). If there's agreement, I'll capture that in a separate ticket for 1.16; I'm limiting this to a hotfix for the UI only.

Screenshot:

<img width="1345" alt="Screen Shot 2024-04-10 at 4 19 08 PM" src="https://github.com/dockstore/dockstore-ui2/assets/1049340/49ae544e-8f46-4d97-9f1d-c4ff51d15450">


**Review Instructions**
1. Navigate to a Nextflow workflow
2. Verify that the Launch with Button says `seqera` instead of `Nextflow Tower`, and has the correct logo.

**Issue**
SEAB-6343

**Security**
If there are any concerns that require extra attention from the security team, highlight them here.

Please make sure that you've checked the following before submitting your pull request. Thanks!
